feat(llma): add evals summary

carlos-marchal-ph opened the pull request:
Adds a small dashboard to evals with a status summary
<img width="1227" height="674" alt="Screenshot 2025-11-11 at 19 40 36" src="https://github.com/user-attachments/assets/2a51ec7c-7a5a-477d-b165-cb4145d1402c" />

carlos-marchal-ph committed:
feat: add evals summary

andrewm4894 commented:
@carlos-marchal-ph one thing im wondering around evals is namings and terms of stuff.

eg if i am setting up an eval to detect unhappy users - the Pass rate is always going to be high typically and its actually the inverse i am interested in focusing on

but then obviously for other evals is more straight forward like did it do the thing - pass rate is what i want 

so its like we maybe need to somehow bear that in mind in the whole UI/UX of evals 

eg:

is_user_unhappy = up is bad
did_bot_work = up is good

sometimes in experiment platforms we have same problem and so users give a hint for the direction or nature of the metric in some way. 

no idea if oe how well all fits in here but just wanted to mention it as we start working on UI and more stuff

"Success Rate" for example is probably a little too inflexible
"Positive Rate" is nice and generic, maybe pas rate too

but there is also something around when a low positive rate is a good thing (eg low amounts of unhappy user flags) vs more natural

and i dont thiknwe can just tell users to invert it all 

as 

is_user_happy != is_user_unhappy when comes to the evals and prompts etc just due to the nature of them - they cant just be inverted like a code test or alert rule etc. 

anyway just wanted to mention all this - not sure there really is any clear good solution

andrewm4894 reviewed (approved):
lgtm - hard to know exaclty what folks might want but best to start with something to help discussions and iterations. 

nice and simple to start in that sense which i like

carlos-marchal-ph commented:
Thanks for the review! This is definitely an MVP, I was playing with it and didn't have much intuition of how to go very deep into anything, so I just added the basics to see how it goes. Nice to see you agree with the approach.

As for the pass/fail thing yep I agree with you that right now it's a bit clunky. It's built expecting true as the positive outcome. For the moment I guess we can do:
- Return true if the user is happy, false if the user is neutral or unhappy.
vs
- Return true if the user is happy or neutral, false if the user is unhappy.


This is not a nice long term solution, so I added [a comment](https://github.com/PostHog/posthog/issues/40241#issuecomment-3522031188) to the ticket about output types to make it customisable. Do you think that would work?

carlos-marchal-ph committed:
fix: address PR feedback

carlos-marchal-ph committed:
fix: missing import statement

github-actions[bot] committed:
test(storybook): update UI snapshots
